pages/data-migration/best-practices.mdx

```diff
@@ -540,6 +540,21 @@ of the `--storage-gc-cycle-sec` flag. That means that Memgraph's garbage
 collector potentially still didn't deallocate unused objects and free the
 memory. You can free up memory by running the `FREE MEMORY` query.
 
+## Monitoring memory during import
+
+Often times, the user needs to batch transactions in the `IN_MEMORY_TRANSACTIONAL` storage mode.
+This is due to large-update transactions that are creating a lot of `Delta` objects during the
+transaction lifetime. `Delta` objects are necessary for reverting the query if an error during 
+query execution happens. They are approximately of 56 bytes in size, but large-update transactions
+can make the number of delta objects be huge - therefore resulting in Memgraph getting out of memory.
+
+We have exposed the `peak_memory_res` variable in the `SHOW STORAGE INFO;` command, with which you
+can monitor when the peak memory rises in the system. It will help you diagnose bottlenecks, and queries
```

Comment:
Awkward. Too many comma. Try something like:
`It will help you diagnose bottlenecks and high-memory queries which can be optimized.`